keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py

```diff
@@ -72,35 +70,41 @@ def get_anchors(
         two together will yield the centerpoints in absolute x,y format.
 
     """
-    base_anchors = tf.constant(base_anchors, dtype=tf.float32)
+    base_anchors = ops.array(base_anchors, dtype="float32")
 
     all_anchors = []
     all_strides = []
     for stride in strides:
-        hh_centers = tf.range(start=0, limit=image_shape[0], delta=stride)
-        ww_centers = tf.range(start=0, limit=image_shape[1], delta=stride)
-        ww_grid, hh_grid = tf.meshgrid(ww_centers, hh_centers)
-        grid = tf.cast(
-            tf.reshape(tf.stack([hh_grid, ww_grid], 2), [-1, 1, 2]),
-            tf.float32,
+        hh_centers = ops.arange(0, image_shape[0], stride)
+        ww_centers = ops.arange(0, image_shape[1], stride)
+        ww_grid, hh_grid = ops.meshgrid(ww_centers, hh_centers)
+        grid = ops.cast(
+            ops.reshape(ops.stack([hh_grid, ww_grid], 2), [-1, 1, 2]),
+            "float32",
+        )
+        anchors = (
+            ops.expand_dims(
+                base_anchors * ops.array([stride, stride], "float32"), 0
+            )
+            + grid
         )
-        anchors = tf.expand_dims(base_anchors * [stride, stride], 0) + grid
-        anchors = tf.reshape(anchors, [-1, 2])
+        anchors = ops.reshape(anchors, [-1, 2])
         all_anchors.append(anchors)
-        all_strides.append(tf.repeat(stride, anchors.shape[0]))
+        all_strides.append(ops.repeat(stride, anchors.shape[0]))
 
-    all_anchors = tf.cast(tf.concat(all_anchors, axis=0), tf.float32)
-    all_strides = tf.cast(tf.concat(all_strides, axis=0), tf.float32)
+    all_anchors = ops.cast(ops.concatenate(all_anchors, axis=0), "float32")
+    all_strides = ops.cast(ops.concatenate(all_strides, axis=0), "float32")
 
     all_anchors = all_anchors / all_strides[:, None]
 
     # Swap the x and y coordinates of the anchors.
-    all_anchors = tf.concat(
-        [all_anchors[:, 1, tf.newaxis], all_anchors[:, 0, tf.newaxis]], axis=-1
+    all_anchors = ops.concatenate(
+        [all_anchors[:, 1, None], all_anchors[:, 0, None]], axis=-1
     )
     return all_anchors, all_strides
 
 
+# TODO: uses conv_bn, csp  => convert to layer
```

Comment:
Can you elaborate on this TODO a bit, and assign it to either yourself or me.

Reply:
Oh, this is outdated now, will remove it!